MANUAL.txt

```diff
@@ -5657,7 +5658,7 @@ and the caption and figure tags will be omitted.
 
 Attributes can be set on links and images:
 
-    An inline ![image](foo.jpg){#id .class width=30 height=20px}
+    An inline ![image](foo.jpg){#id .class width=30 height=20px alt="Alt text"}
```

Comment:
This is not a good example, because in this inline image, the alt text will be set from the image description. Thus, you can just do `![alt text](foo.jpg)`.  You would only have a reason to use `alt=` when the image is by itself in a paragraph, when the `implicit_figures` extension is set.